### ISSUE TITLE
IBM Terraform Future Features

### ISSUE THREAD

**Issue by agrare** [ManageIQ/manageiq-providers-ibm_terraform#60]
Event management between ManageIQ and IBM Terraform

- [ ] Initiate targeted refresh when the terraform deployments finish
- [ ] Initiate terraform refresh/action when virtual machines are changed in ManageIQ (requires design discussion)

**Comment by chessbyte** [comment]
was anything done in this area for the Oparin release?

**Comment by agrare** [comment]
No, and given we've bumped this back two releases already probably best to drop the target release from this and leave it just as "future enhancements"
